backend/accounts/baseaccount.go

```diff
@@ -245,6 +246,7 @@ func (account *BaseAccount) ExportCSV(w io.Writer, transactions []*TransactionDa
 		"Amount",
 		"Unit",
 		"Fee",
+		"Fee Unit",
```

Comment:
Might adding a column break Cointracking? @Beerosagos

Reply:
I asked them

Reply:
They replied it's fine :+1: